src/libData/AccountData/AccountStore.cpp

```diff
@@ -86,8 +86,6 @@ void AccountStore::Init() {
 }
 
 void AccountStore::InitSoft() {
-  LOG_MARKER();
```

Comment:
Are you sure removing those LOG_MARKERs is safe?  Surely don't have to do this to bring Eth-style transactions in.

Reply:
I did not consider that this might not be safe - I thought that they just print 'BEG' and 'END' according to its lifetime (and so only useful if you are debugging that function, it is just generic cleanup). In what way might it not be safe?

Reply:
Agree useful to just keep the logs